README.md

```diff
@@ -58,85 +58,38 @@ The version listed is used in current environment. This must not be identical to
     ```
     Error: Cannot find module 'generate'
     ```
-    - when you execute *local test* via `prisma deploy` command with `prisma generate` at **post-deploy hook setting** in `prisma.yml`, this error occured in prisma *Homebrew installation*.
+    - when you execute *local test* via `prisma deploy` command at **post-deploy hook setting** in `prisma.yml`, this error occured in prisma *Homebrew installation*.
     - that error not fixed in *Homebrew installation* yet.
     - **so install via npm**
     - [GitHub issues: Error: Cannot find module 'generate'](https://github.com/prisma/Mongo-Connector-Preview/issues/1#issuecomment-441361867)
    
 ### Running server
-1. `$ docker-compose up -d`
```

Comment:
@geoseong Your updates has been deleted by this `PR`. Please confirm!

Reply:
because we've dropped the prisma's API binding (for now)

Reply:
@hyochan @cometkim okay. no problem